### DISCUSSION TITLE
How do I turn off the time limit?

### DISCUSSION THREAD

**Discussion by MasterGlitcher2006** [Techokami/SonicWorldsNext#209] [Q&A]
How do I make it so that then 10 minutes have passed, the character hasn't died?

**Comment by Caverns4** [comment]
in Global.gd:
`var maxTime = 60*10`
Change that to the time you want, or remove the limit altogether in player.gd
`	# Time over
	if Global.levelTime >= Global.maxTime:
		kill(true)`

**Reply by MasterGlitcher2006** [reply]
It works, but after that the death.wav sound plays endlessly.

**Reply by Caverns4** [reply]
That does not happen on my end. You DID also get rid of the "kill(true)", right?
